bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -1015,12 +1015,13 @@ bool ReplicaImp::validatePreProcessedResults(const PrePrepareMsg *msg, const Vie
       const MessageBase::Header *hdr = (MessageBase::Header *)requestBody;
       if (hdr->msgType == MsgCode::PreProcessResult) {
         tasks.push_back(threadPool.async(
-            [&errors, requestBody, error_id](auto replicaId, auto fVal) {
+            [&errors, requestBody, error_id](auto replicaId, auto fVal, auto viewNum) {
```

Comment:
Did you touch PreProcessResultMsg? We decided not to touch it for now. Please revert.